s3transfer/manager.py

```diff
@@ -40,8 +41,8 @@ def __init__(self,
                  multipart_chunksize=8 * MB,
                  max_request_concurrency=10,
                  max_submission_concurrency=5,
-                 max_request_queue_size=0,
-                 max_submission_queue_size=0,
+                 max_request_queue_size=1000,
+                 max_submission_queue_size=1000,
```

Comment:
Yeah just to protect the users. Also since we are using semaphores, a value of zero is strangely acceptable but causes deadlocks if you try to acquire it.